tests/TraceContext/W3CTestService/index.php

```diff
@@ -25,25 +30,33 @@
     API\SpanKind::KIND_INTERNAL
 );
 
+$httpClient = new Psr18Client();
+
 $exporter = new JaegerExporter(
     'W3C Trace-Context Test Service',
-    'http://localhost:9412/api/v2/spans'
+    'http://localhost:9412/api/v2/spans',
+    $httpClient,
+    $httpClient,
+    $httpClient,
 );
 
-if (SamplingResult::RECORD_AND_SAMPLE === $samplingResult->getDecision()) {
-    $tracer = (new TracerProvider())
-        ->addSpanProcessor(new BatchSpanProcessor($exporter, ClockFactory::getDefault()))
+$shouldRecordAndSampleSpan = $samplingResult->getDecision() === SamplingResult::RECORD_AND_SAMPLE;
```

Comment:
The sampling decision is part of SpanBuilder now, and is taken care of automatically.

Reply:
Thanks. I'll change this just as we're documenting in #689.